ArduPlane/quadplane.cpp

```diff
@@ -1728,11 +1744,18 @@ void QuadPlane::update_transition(void)
         }
         hold_hover(climb_rate_cms);
 
-        // set desired yaw to current yaw in both desired angle and
-        // rate request. This reduces wing twist in transition due to
-        // multicopter yaw demands
-        attitude_control->set_yaw_target_to_current_heading();
-        attitude_control->rate_bf_yaw_target(ahrs.get_gyro().z);
+        if (tilt.tilt_mask == 0 ||
+            tilt.tilt_type != TILT_TYPE_VECTORED_YAW ||
+            now - transition_start_ms < 100) {
+            // set desired yaw to current yaw in both desired angle
+            // and rate request. This reduces wing twist in transition
+            // due to multicopter yaw demands. This is disabled when
+            // using vectored yaw for tilt-rotors as the yaw control
+            // is needed to maintain good control in forward
+            // transitions
```

Comment:
I wonder if there should be a `Q_OPTION` for this, I suspect some quadplanes could handle the torque fine.

Reply:
We originally used a similar method to disable yaw torque before creating the new frame types, but I can't remember why we changed.

Reply:
once you have tilt for yaw, torque really makes no sense, and it really screws things up badly once you are tilted.

Reply:
I was thinking of a none tilting quad-plane. No reason they cant use yaw too if they are can handle the wing twist mentioned in the comment.